docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# This docker-compose file will deploy the latest tagged images from ghcr.io,
+# which are built and pushed whenever a new Git tag is pushed to GitHub.
+# Use this file to deploy ODK Build to a staging or production server.
+# Run: docker-compose up -d
+version: "3"
+services:
+  postgres:
+    image: "postgres:14.1"
+    volumes:
+      - /var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: odkbuild
+      POSTGRES_PASSWORD: odkbuild
+      POSTGRES_DATABASE: odkbuild
+    restart: always
+  build2xlsform:
+    # image: "ghcr.io/getodk/build2xlsform:latest"
+    image: "ghcr.io/florianm/odkbuild2xlsform:21-docker"
```

Comment:
assuming this is temp

Reply:
Yep, pending the corresponding PR there. Chicken and egg. This will be test/prof ready once both Build and build2xlsform support the same feature set, and both are built under the getodk ghcr docker image repo. 

The optimal naming and tagging schemes need to be worked out still. I felt an "odk" prefix would be good to distinguish the software ODK Build against the verb and process "build".